.github/pull_request_template.md

```diff
@@ -13,6 +13,8 @@ of the PR were done in a specific way -->
 
 <!-- Notice the release manager should include in the release tag message changelog -->
 <!-- See https://keepachangelog.com/en/1.0.0/ for examples -->
+* [ ] I've added a changelog in the next release tracking issue (see [example](https://github.com/bitcoindevkit/bdk-ffi/issues/875))
```

Comment:
Let's put it under the `### Checklist` heading what do you think?

Reply:
And then we can remove the whole `Changelog Notice` heading altogether, hoping people add it to the issue instead.

Reply:
@thunderbiscuit pr is updated with those suggested changes now